runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/Tree.java

```diff
@@ -28,12 +28,14 @@
 import java.util.Map;
 import lombok.Getter;
 import static net.runelite.api.ObjectID.REDWOOD;
+import static net.runelite.api.ObjectID.REDWOOD_29669;
 import static net.runelite.api.ObjectID.REDWOOD_29670;
+import static net.runelite.api.ObjectID.REDWOOD_29671;
 
 @Getter
 enum Tree
 {
-	REDWOOD_TREE_SPAWN(REDWOOD, REDWOOD_29670);
+	REDWOOD_TREE_SPAWN(REDWOOD, REDWOOD_29669, REDWOOD_29670, REDWOOD_29671);
```

Comment:
This should be changed to:  
REDWOOD_TREE_SPAWN(REDWOOD, REDWOOD_29670, NULL_34633, NULL_34635, NULL_34637, NULL_34639);

REDWOOD_29669 and REDWOOD_29671 are the depleted versions.